packages/react/emotion-utils/src/FullHeight.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from '@testing-library/react';
+import { FullHeight } from './FullHeight';
+
+const setInnerHeight = (height: number) => {
+  Object.defineProperty(window, 'innerHeight', { writable: true, configurable: true, value: height });
+};
+
+const CHILDREN_TEXT = 'FullHeight';
+
+describe('FullHeight', () => {
+  it('should have height 768px', () => {
+    render(<FullHeight>{CHILDREN_TEXT}</FullHeight>);
+
+    const fullHeightBox = screen.getByText(CHILDREN_TEXT);
+
+    expect(window.innerHeight).toBe(768);
+
+    expect(fullHeightBox).toBeInTheDocument();
+    expect(fullHeightBox).toHaveStyle('height: 768px');
+  });
+
+  it('should have height 1024px', () => {
```

Comment:
Could you write two different test descriptions differently😄

One is to follow the default height, and the other is to follow the height that you set, 
so what test codes want to verify is different

Reply:
@okinawaa 
Thank you🙏 I fixed the code again.